Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -1954,35 +1954,73 @@ WarpXParticleContainer::GetDebyeLength (int lev)
     return debye_length;
 }
 
-amrex::ParticleReal WarpXParticleContainer::sumParticleWeight(bool local) {
+std::pair<amrex::ParticleReal, amrex::ParticleReal> WarpXParticleContainer::sumParticleWeightAndEnergy(bool local) const {
```

Comment:
Suggesting to leave a white space between the function's name and the list of arguments, following https://warpx.readthedocs.io/en/latest/developers/contributing.html#style-and-conventions:
```suggestion
std::pair<amrex::ParticleReal, amrex::ParticleReal> WarpXParticleContainer::sumParticleWeightAndEnergy (bool local) const {
```